build.cake

```diff
@@ -103,111 +99,17 @@ Task("GetAddinPackages")
-Task("Preview")
-    .IsDependentOn("GetAddinSpecs")
-    .Does(() =>
-    {
-        Wyam(new WyamSettings
-        {
-            Recipe = "Docs",
-            Theme = "Samson",
-            UpdatePackages = true,
-            Preview = true,
-            Watch = true,
-            PreviewVirtualDirectory = "Cake.Prca.Website",
-            Settings = new Dictionary<string, object>
-            {
-                { "AssemblyFiles",  addinSpecs.Where(x => x.Assemblies != null).SelectMany(x => x.Assemblies).Select(x => "../release/addins" + x) },
-                { "Host",  "cake-contrib.github.io" },
-                { "LinkRoot",  "Cake.Prca.Website" },
-                { "BaseEditUrl", "https://github.com/cake-contrib/Cake.Prca.Website/tree/develop/docs/input/" },
-            }
-        });
-    });
-
-// Assumes Wyam source is local and at ../Wyam
-Task("Debug")
-    .Does(() =>
-    {
-        StartProcess("../Wyam/src/clients/Wyam/bin/Debug/wyam.exe",
-            "-a \"../Wyam/src/**/bin/Debug/*.dll\" -r \"docs -i\" -t \"../Wyam/themes/Docs/Samson\" -p --attach");
-    });
-
-Task("Deploy")
-    .WithCriteria(isRunningOnAppVeyor)
-    .WithCriteria(!isPullRequest)
-    .WithCriteria(isMasterBranch)
-    .WithCriteria(!string.IsNullOrEmpty(accessToken))
-    .WithCriteria(!string.IsNullOrEmpty(deployRemote))
-    .WithCriteria(!string.IsNullOrEmpty(deployBranch))
-    .IsDependentOn("Build")
-    .Does(() =>
-    {
-        EnsureDirectoryExists(rootPublishFolder);
-        var sourceCommit = GitLogTip("./");
-        var publishFolder = rootPublishFolder.Combine(DateTime.Now.ToString("yyyyMMdd_HHmmss"));
-        Information("Getting publish branch {0}...", deployBranch);
-        GitClone(deployRemote, publishFolder, new GitCloneSettings{ BranchName = deployBranch });
-
-        Information("Sync output files...");
-        Kudu.Sync(outputPath, publishFolder, new KuduSyncSettings {
-            PathsToIgnore = new []{ ".git", "appveyor.yml" }
-        });
-
-        Information("Stage all changes...");
-        GitAddAll(publishFolder);
-
-        Information("Commit all changes...");
-        GitCommit(
-            publishFolder,
-            sourceCommit.Committer.Name,
-            sourceCommit.Committer.Email,
-            string.Format("AppVeyor Publish: {0}\r\n{1}", sourceCommit.Sha, sourceCommit.Message)
-            );
-
-        Information("Pushing all changes...");
-        GitPush(publishFolder, accessToken, "x-oauth-basic", deployBranch);
-    });
-
-//////////////////////////////////////////////////////////////////////
-// TASK TARGETS
-//////////////////////////////////////////////////////////////////////
-
-Task("Default")
-    .IsDependentOn("Build");
-
 Task("GetArtifacts")
     .IsDependentOn("GetAddinPackages");
 
-Task("AppVeyor")
-    .IsDependentOn(!isPullRequest && isMasterBranch ? "Deploy" : "Build");
+BuildParameters.Tasks.BuildDocumentationTask
```

Comment:
@pascalberger I was going to ask if you knew about this extension point in Cake.Recipe for adding custom functionality :+1: